libnmstate/netapplier.py

```diff
@@ -341,3 +344,11 @@ def _cleanup_iface_ipv6_link_local_addr(desired_state, current_state):
             addr_str = '%s/%d' % (addr_dict['ip'], addr_dict['prefix-length'])
             if nm.ipv6.is_link_local_addr(addr_str):
                 state['ipv6']['address'].remove(addr_dict)
+
+
+def _sort_ip_addresses(desired_state, current_state):
```

Comment:
It will fit better outside this module, perhaps to an ipaddress module. We can delay this to an overall cleanup of netapplier.